src/main/java/ro/msg/learning/shop/strategy/SingleLocationStrategy.java

```diff
@@ -0,0 +1,18 @@
+//package ro.msg.learning.shop.strategy;
```

Comment:
Remove the commented code if you don't need it now. As I said in the previous PR, you should never push commented code to git